src/components/Auth/VerifyEmail.js

```diff
@@ -0,0 +1,25 @@
+import React, { useEffect } from 'react';
+import axios from 'axios';
+import { Link } from 'react-router-dom';
+import { config } from '../../helpers/constants';
+
+const VerifyEmail = () => {
+  const urlParams = new URLSearchParams(window.location.search);
+  const key = urlParams.get('key');
+
+  useEffect(() => {
+    axios.post(`${config.API_URL}/api/v1/auth/registration/verify-email/`, {
+      key,
+    });
+  });
+
+  return (
+    <div>
+      <div>
+        email verified! <Link to="/">go back home</Link>
```

Comment:
Instead of always showing the "Email verified!" text, we can show it only if the status code was 200. (And show an error if the POST request did not work.) SeeL comment above